tests/foreman/ui/test_org.py

```diff
@@ -367,34 +375,30 @@ def test_remove_domain(self, domain_name):
         """
         strategy, value = common_locators['entity_select']
         strategy1, value1 = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        domain = entities.Domain(name=domain_name).create()
-        self.assertEqual(domain.name, domain_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name, domains=[domain_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_domains'])
-            element = session.nav.wait_until_element((strategy1,
-                                                      value1 % domain_name))
-            # Item is listed in 'Selected Items' list and not 'All Items' list.
-            self.assertIsNotNone(element)
-            self.org.update(org_name, domains=[domain_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_domains'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % domain_name))
-            # Item is listed in 'All Items' list and not 'Selected Items' list.
-            self.assertIsNotNone(element)
-
-    #  Note: HTML username is invalid as per the UI msg.
-    @data(
-        gen_string('alpha'),
-        gen_string('numeric'),
-        gen_string('alphanumeric'),
-        gen_string('utf8'),
-        gen_string('latin1'),
-    )
-    def test_remove_user(self, user_name):
+            for domain_name in generate_strings_list():
```

Comment:
Not a blocker but I was thinking... wouldn't make more sense to create only one organization and then associate a newly created domain (and then remove it) inside the loop? If we already have a test where an organization is created with a domain associated to it (which is a valid test; heck, even creating a new organization and add multiple domains at creation is another valid test hehe), maybe we can speed things up a bit here by not creating a new org for every iteration?